front-end/src/api/apiService.tsx

```diff
@@ -0,0 +1,32 @@
+import { Student } from '../redux/slice';
+import axios from 'axios';
+
+export const getAllStudentsFromApi = async () => {
+  const response = await axios.get(`http://localhost:4000/students`);
+  return response.data as Student[];
+};
+
+export const deleteStudentApi = async (studentId: number) => {
+  await axios.delete(`http://localhost:4000/students/${studentId}`);
```

Comment:
you can create Axios instance and use that instance, when you do that you don't need to set the base URL every place